config/environments/test.rb

```diff
@@ -8,12 +8,13 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Turn false under Spring and add config.action_view.cache_template_loading = true.
   config.cache_classes = true
 
-  # Do not eager load code on boot. This avoids loading your whole application
-  # just for the purpose of running a single test. If you are using a tool that
-  # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  # Eager loading loads your whole application. When running a single test locally,
+  # this probably isn't necessary. It's a good idea to do in a continuous integration
+  # system, or in some way before deploying your code.
+  config.eager_load = ENV["CI"].present?
```

Comment:
I wonder if there is a speed benefit if eager_load is set to true in our CI environments (doesn't really matter for this app as it's tiny there's like 3 tests)

Reply:
Don't know - this is the default change put in by the rails update, so per the process I think we're supposed to keep it unless it's going to cause trouble. I assume so we keep as close to base Rails 7 config as possible.